### PR TITLE
Updated delay strategies to take TimeSpan in the constructor instead of m…

### PR DESCRIPTION
**Pull request by vivekse**
Updated delay strategies to take TimeSpan in the constructor instead of minutes

### CODE REVIEW

File: src/MooseSoft.Azure.ServiceBus/BackOffDelayStrategy/LinearBackOffDelayStrategy.cs [hunk 1]
```diff
@@ -8,19 +8,19 @@ namespace MooseSoft.Azure.ServiceBus.BackOffDelayStrategy
...
     public class LinearBackOffDelayStrategy : IBackOffDelayStrategy
     {
-        private readonly int _multiplier;
+        private readonly TimeSpan _multiplier;
 
-        public LinearBackOffDelayStrategy(int multiplier)
+        public LinearBackOffDelayStrategy(TimeSpan multiplier)
         {
             _multiplier = multiplier;
         }
```
Comment (gtmoose32) [inline src/MooseSoft.Azure.ServiceBus/BackOffDelayStrategy/LinearBackOffDelayStrategy.cs]: A multiplier as a time span really doesn't make sense.  The idea here was to set a multiplier that will scale up by the number of attempts.  The name might not be the best for this but I'd remove these changes.

File: src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
```
Comment (gtmoose32) [inline src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj:6]: Modifying param types of existing ctors without support backwards compatibility is a breaking change.  Therefore, the version needs to reflect that or add support for backwards compatible ctors.

**Review by gtmoose32 — COMMENTED** [review]
Made a few comments.

File: src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj (hunk 2 above)
Comment (vivekse) [reply src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj:6]: added the old constructors back for backward compatibility.

File: src/MooseSoft.Azure.ServiceBus/BackOffDelayStrategy/LinearBackOffDelayStrategy.cs (hunk 1 above)
Comment (vivekse) [reply src/MooseSoft.Azure.ServiceBus/BackOffDelayStrategy/LinearBackOffDelayStrategy.cs]: reverted these changes.

File: src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj (hunk 2 above)
Comment (vivekse) [reply src/MooseSoft.Azure.ServiceBus/MooseSoft.Azure.ServiceBus.csproj:6]: Added the old constructors back for backward compatibility

**Comment by vivekse** [comment]
Addressed the review comments, let me know how it looks now.

**Pull request merged**